source/photo_audio_video_file.md

```diff
@@ -0,0 +1,148 @@
+# Photo, audio, video and file question types
+
+With KoboToolbox, you can collect different types of media as part of your data
+collection project.
+
+When you want to capture images as part of your submissions, use the “Photo”
+question type.
+
+If a question requires that you record or attach an audio file, such as when a
+long explanation is expected from the respondent, use the “Audio” question type.
+The latest version of KoboCollect allows you to record audio within the app
+itself without opening a separate app.
+
+With the “Video” question type, you will be able to record a video using the
+device camera or attach a video file. This may be particularly useful when a
+question requires the respondent to demonstrate something.
+
+If a question requires that you attach a file such as a PDF, you can use the
+“File” question type.
+
+Note: Your device should also have an in-built camera and audio recorder to use
+the "photo", "audio" and "video" question types while collecting data..
+
+## How to set up "photo", "audio", "video" and "file" question types
+
+### Setting up in FormBuilder
+
+Adding media questions on the form is easy and does not require any extra setup:
+
+-   In the KoboToolbox Formbuilder, click the + button to add a new question
+-   Type the question text, for example “Take a picture of the housing unit”,
+    then click ADD QUESTION or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding media question](images/photo_audio_video_file/add.gif)
+
+### Setting up in XLSForm
```

Comment:
It might also be nice to have some Excel files for people to download with these XLSForms 👌

Reply:
Added